src/instructlab/configuration.py

```diff
@@ -413,6 +414,7 @@ def get_default_config() -> Config:
             mmlu=_mmlu(
                 few_shots=2,
                 batch_size=5,
+                device="cpu"
```

Comment:
Are we not going to be able to do autodetection?

Reply:
Train and evaluate device support are going in separately. I'm thinking device autodetection could go in a follow-up PR.

Reply:
I've also been hearing the idea of using different devices for different portions of ilab (train on one device and eval on another), so we may not want to limit to one device all the way through? But maybe autodetection could happen separately (autodetection at train and another autodetection at eval).